corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -312,8 +312,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
 
             $.when(FormplayerFrontend.getChannel().request("icon:click", currentUrlToObject)).done(function () {
-                self.reloadCase(self.model.get('id'));
-                resetIcon();
+                setTimeout(() => {
+                    self.reloadCase(self.model.get('id'));
+                    resetIcon();
+                }, 500);
```

Comment:
half a second is enough to account for ES lag?

Reply:
In QA there was a ~50% success rate with getting the accurate case change before calling `get_details`, so I think based on that, this was the estimation of what would/could work. 
@shubham1g5 correct me if I'm missing anything. 
I'll be testing on staging and we could increase it if needed.

Reply:
Interesting, okay. With this marked as a temporary fix, is there a different long-term plan?

Reply:
My understanding is that we'll need to create a new ticket to investigate further how case search is interacting with clickable icons. The 50% success/failure rate is when case search is used in conjunction with clickable icons.

Reply:
okay, got it

Reply:
That's right. This limit is definitely a rough limit at this moment and is not going to work in all cases for eg. when ES is non performant due to pillow lag or any other infra issues. It's more of a temporary solution to probably try and address 90% of the cases but we definitely expect this to come up as a bug pretty soon.